virt/virt-4-10-release-notes.adoc

```diff
@@ -234,7 +242,53 @@ metadata:
+    "path": "/spec/kubeMacPool"
+    "value": null
+  }
+ ]'
+----
++
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2035344[*BZ#2035344*])
+
+* {VirtProductName} links a service account token in use by a pod to that specific pod. `Kubevirt` implements a `ServiceAccountVolume` by creating a disk image that contains a token. If you migrate a VM, then the `ServiceAccountVolume` becomes invalid.
+** As a workaround, use user accounts rather than service accounts because user account tokens are not bound to a specific pod. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2037611[*BZ#2037611*])
+
+* You cannot attempt to stop a VM multiple times because `KubeVirt` prevents multiple stop attempts. If a VM crashes during shutdown, then you cannot issue a new stop attempt and you cannot easily remove the VM from the UI. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2040766[*BZ#2040766*])
+
+* If you configure the `HyperConverged` custom resource (CR) to enable mediated devices before drivers are installed, then configuration setting changes to the CR will not be saved. This issue can be triggered by updates. For example, if `virt-handler` is updated before `daemonset`, which installs NVIDIA drivers, then nodes are not able to provide virtual machine GPUs.
+** As a workaround:
+. Remove `mediatedDevicesConfiguration` and `permittedHostDevices` from the `HyperConverged` CR.
+. Update both `mediatedDevicesConfiguration` and `permittedHostDevices` stanzas with the desired configuration.
++
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2046298[*BZ#2046298*])
+
+* YAML examples are not always up to date with the latest information because the examples are hardcoded. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2055492[*BZ#2055492*])
+
+* A non-privileged user cannot use the `Add Network Interface` button when creating a VM due to role-based access control (RBAC) rules.
+** As a workaround, non-privileged users can watch the `NAD Network Settings` screen to confirm the VM creation process is successful. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2056420[*BZ#2056420*])
+
+* A non-privileged user cannot add disks to a VM due to RBAC rules.
+** As a workaround, manually add the RBAC rule to allow specific users to add disks. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2056421[*BZ#2056421*])
+
+* The web console does not display virtual machine templates that are deployed to a custom namespace. Only templates deployed to the default namespace will display in the web console.
+** As a workaround, avoid deploying templates to a custom namespace. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2054650[*BZ#2056623*])
+
+* If you clone more than 100 VMs using the `csi-clone` cloning strategy, then the Ceph CSI might not purge the clones. Manually deleting the clones might not work either.
+** As a workaround, you can restart the `ceph-mgr` to purge the VM clones. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2060371[*BZ#2060371*])
```

Comment:
Bugzilla 2055595 should be referenced here

Reply:
Can you please add Doc Text to Bug 2055595 so I can include it?

Reply:
> @dominikholler Can you please add Doc Text to Bug 2055595 so I can include it?

@mgarrellRH This would be not correct. RHBZ 2060371 is the know issue about what RHBZ 2055595 means for OpenShift Virtualization, while 2055595 is about a bug in ceph, which is independent from OpenShift Virtualization.